lib/internal/fs/promises.js

```diff
@@ -190,9 +190,8 @@ async function readFileHandle(filehandle, options) {
   const result = Buffer.concat(chunks);
   if (options.encoding) {
     return result.toString(options.encoding);
-  } else {
-    return result;
   }
+  return result;
```

Comment:
This can be further simplified to:

```js
  return options.encoding ? result.toString(options.encoding) : result;
```

Reply:
Thanks @jasnell  for the feedback, I will make this change right now.